akka-http-core/src/main/resources/reference.conf

```diff
@@ -260,6 +260,21 @@ akka.http {
       # be increased for high bandwidth-delay-product connections.
       incoming-stream-level-buffer-size = 512kB
 
+      # For incoming requests, the infrastructure collects at least the given number of bytes before dispatching a HttpRequest.
+      # If all request data is received before or when the threshold is reached, the entity data is dispatched as a strict entity
+      # which allows more efficient processing of the request data without involving streams.
+      #
+      # If the given value is 0, the request is immediately dispatched and always carries a stream of data.
+      # You can use value `1` to create a strict entity if the request contains at most a single data frame, and a streamed
+      # entity otherwise.
+      #
+      # If a strict entity is dispatched, the data size will be at least the amount given here.
```

Comment:
If the HEADERS has `endStream` set, don't we dispatch an empty 'strict' entity?

Reply:
Yes, this is misleading, I just removed the sentence.